configs/generic_x86-64_hdd_config

```diff
@@ -18,9 +18,9 @@
 #
 # Converted with the following command:
 #
-#	MENDER_ARTIFACT_NAME=release-1 ./docker-mender-convert \
-#		--disk-image input/centos.img \
-#		--config configs/centos_x86-64_hdd_config \
+#	MENDER_ARTIFACT_NAME=release-1 ./docker-mender-convert
+#		--disk-image input/centos.img
+#		--config configs/centos_x86-64_hdd_config
```

Comment:
This is wrong, the comment is incorrect now. Not a huge deal, I know, but can we make it ignore comments completely?

Reply:
Yeah, I know. I'll investigate a bit more.

Reply:
I found a way to disabling formatting on this part of the file. See upcoming new commit.